unime/src-tauri/tests/fixtures/states/three_connections_search_delete_recent.json

```diff
@@ -33,5 +33,9 @@
          "last_interacted":"2023-11-08T13:52:45.868117252+00:00"
       }
     ],
-    "user_data_query": ["Gym XYZ", "Government"]
+    "search_results": {
+      "current": ["Government", "Gym XYZ"],
+      "recents_connections": [],
```

Comment:
Outdated `"recents_connections"`, also rename `"recents_credentials"` to `"recent_credentials"`